rocky/tools/forms/ooi_form.py

```diff
@@ -109,25 +106,35 @@ def generate_select_ooi_field(
     field: FieldInfo,
     related_ooi_type: type[OOI],
     initial: str | None = None,
-) -> forms.fields.Field:
+) -> forms.fields.Field | None:
     # field is a relation, query all objects, and build select
     default_attrs = default_field_options(name, field)
     is_multiselect = getattr(field.annotation, "__origin__", None) is list
     option_label = default_attrs.get("label", _("option"))
 
-    option_text = "-- " + _("Optionally choose a {option_label}").format(option_label=option_label) + " --"
+    manytext = "one or more:" if is_multiselect else "a"
     if field.is_required():
-        option_text = "-- " + _("Please choose a {option_label}").format(option_label=option_label) + " --"
+        option_text = _("Please choose {manytext} {option_label}").format(option_label=option_label, manytext=manytext)
+    else:
+        option_text = _("Optionally choose {manytext} {option_label}").format(
+            option_label=option_label, manytext=manytext
+        )
```

Comment:
I think using `django.utils.translation.ngettext` would be better. I'm not sure if in the current implementation the `many_text` variable will be translated correctly